roles.json

```
@@ -166,6 +166,22 @@
                         "Identify opportunities for workshops in diverse geographic locations"]
         }
     },
+    {
+        "role": "Affiliated package review editor",
+        "url": "Affiliated_package_review_editor",
+        "lead": ["Pey-Lian Lim", "Hans Moritz G\u00fcnther"],
```

Hyphen is not official.
```suggestion
        "lead": ["Pey Lian Lim", "Hans Moritz G\u00fcnther"],
```

Also, how are these names ordered? Either by first or last name, @hamogu should have been listed first.

---

I saw that, too, and concluded that it's probably reverse alphabetically ordered (either by first of by last name, could not figure that out)  ;-)

---

it was in random order ;) I just re-ran with a new random seed and got alphabetical, so I'll go with that!